internal/types/documents_iterator.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import "github.com/FerretDB/FerretDB/internal/util/iterator"
+
+// DocumentsIterator represents an iterator over documents.
```

Comment:
Should we specify what `key` means in this case?

Reply:
That's undefined in the type itself, and documented by users of it